pkg/kubelet/kubelet.go

```diff
@@ -188,6 +189,18 @@ const (
 
 	// nodeLeaseRenewIntervalFraction is the fraction of lease duration to renew the lease
 	nodeLeaseRenewIntervalFraction = 0.25
+
+	// kubeletInstrumentationScope is the name of OpenTelemetry instrumentation scope
+	// for this package
+	kubeletInstrumentationScope = "k8s.io/kubernetes/pkg/kubelet"
+
+	// imagesInstrumentationScope is the name of OpenTelemetry instrumentation scope
+	// for images subpackage
+	imagesInstrumentationScope = "k8s.io/kubernetes/pkg/kubelet/images"
```

Comment:
nit: Only the pkg/kubelet scope should be defined in pkg/kubelet. The kubelet should pass a `TracerProvider` to `NewKubeGenericRuntimeManager` and `NewImageGCManager` like you had before, and the `Tracer` should be created in those New* functions.

Reply:
Good idea